### ISSUE TITLE
Old package version in Debian Unstable

### ISSUE THREAD

**Issue by MasFlam** [dankamongmen/notcurses#2738]
Hey, it's the tabbed guy again coming back to notcurses after a while with a question. Is there a reason the package version on Debian Unstable is held back at 3.0.7? If it's related to the doctest v2.4.9 bug, it shouldn't be a problem anymore since sid's doctest is now 2.4.11

**Comment by MasFlam** [comment]
I'm actually asking cause I decided to finally learn me some rust and I can't get any 3.x notcurses-rs version to build because all of them require notcurses 3.0.8 or 3.0.9 :/ Building and installing 3.0.9 from the github release tarball still has the cargo build of `libnotcurses-sys` not like me, weirdly enough.

**Comment by dankamongmen** [comment]
indeed. i uploaded it, but the tests didn't run successfully for some big-endian architecture, and i need to get them fixed before it'll be allowed to proceed through to unstable =[. i think i have a bug open on this; let me look for it.

**Comment by dankamongmen** [comment]
or maybe i don't? i've got a lengthy email from paul gevers (iirc) from last march that goes into what's wrong. i'll get into it in the new year, when i intend to come back to notcurses.

**Comment by MasFlam** [comment]
I could take a look at it myself and try fixing the regression if you don't mind. Are those tests run in some CI/CD thingy that I can see the results of online?
